src/sql/src/plan.rs

```diff
@@ -564,23 +564,27 @@ impl QueryWhen {
         }
     }
     /// Returns whether the candidate must be advanced to the since.
-    pub fn advance_to_since(&self) -> bool {
+    pub fn advance_to_since(&self, strict_serializable: bool) -> bool {
```

Comment:
Should these methods take a `IsolationLevel` instead of a `bool`? That might be a bit cleaner.

Reply:
yes!

Reply:
Although I'm now doubting that these methods should take arguments at all. Should the bool checks be done in determine_timestamp? I don't know.